### PR TITLE
⬆️ Upgrade Android Dependencies & Flutter Embedding

### PR DESCRIPTION
**Pull request by AlejandroHCruz**
**Changes**
* added support for Android Flutter EmbeddingV2, keeping support for V1 as recommended in the Flutter guidelines
* updated Adyen SDK Android to 4.4.0
* removed jCenter dependency by replacing it for MavenCentral
* bumped version to v.0.10.0

**Notes**
- Please note that this branch was based off the `null safety` one.
- The test application compiles and runs
- I see now that master has bumped the version to 0.9.0 already, so in this branch I'm bumping to 0.10.0 if you also agree

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -26,27 +26,27 @@ buildscript {
 
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```
Comment (AlejandroHCruz) [inline android/build.gradle:29]: `jcenter` is deprecated, in our case, `mavenCentral` was a direct replacement

File: android/build.gradle [hunk 2]
```diff
@@ -26,27 +26,27 @@ buildscript {
...
     }
 }
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 31
```
Comment (AlejandroHCruz) [inline android/build.gradle:49]: Flutter does not yet support Android API 32

File: example/ios/Flutter/flutter_export_environment.sh [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
 export "FLUTTER_BUILD_DIR=build"
 export "FLUTTER_BUILD_NAME=1.0.0"
 export "FLUTTER_BUILD_NUMBER=1"
-export "DART_DEFINES=Zmx1dHRlci5pbnNwZWN0b3Iuc3RydWN0dXJlZEVycm9ycz10cnVl,RkxVVFRFUl9XRUJfQVVUT19ERVRFQ1Q9dHJ1ZQ=="
 export "DART_OBFUSCATION=false"
-export "TRACK_WIDGET_CREATION=true"
+export "TRACK_WIDGET_CREATION=false"
 export "TREE_SHAKE_ICONS=false"
-export "PACKAGE_CONFIG=/Users/benjaminbaumann/projects/adyen_flutter/example/.dart_tool/package_config.json"
+export "PACKAGE_CONFIG=.packages"
```
Comment (AlejandroHCruz) [inline example/ios/Flutter/flutter_export_environment.sh:13]: ⚠️ Shouldn't we remove this file from git?

File: pubspec.lock [hunk 4]
```diff
@@ -0,0 +1,147 @@
+# Generated by pub
```
Comment (AlejandroHCruz) [inline pubspec.lock:1]: I'm not sure why I've committed this file if it was not being tracked before, I'll remove it if you agree.

File: pubspec.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 name: adyen_dropin
 description: Flutter plugin to integrate Adyen DropIn UI for Android and iOS.
-version: 0.8.0
+version: 0.9.1
```
Comment (AlejandroHCruz) [inline pubspec.yaml:3]: Should probably be bumped to v0.10.0, as `master` has deviated into v0.9.0 already 

- [ ] change

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+## 0.10.0
+* updated Adyen SDK Android to 4.4.0
+* removed jCenter dependency by replacing it for MavenCentral
+* added support for Android Flutter EmbeddingV2
+
```
Comment (AlejandroHCruz) [inline CHANGELOG.md:5]: v0.9.0 is in master

File: android/build.gradle [hunk 7]
```diff
@@ -62,16 +62,15 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.adyen.checkout:drop-in:3.8.2"
-    implementation "com.adyen.checkout:card-ui:3.8.2"
```
Comment (AlejandroHCruz) [inline android/build.gradle:66]: 🔥 Removed, was apparently not being used.

**Review by aignerjo — APPROVED** [review]

**Pull request merged**